### PR TITLE
Rename and remove dir integration test for local files

### PR DESCRIPTION
**Pull request by ashmeenkaur**
### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual 
- Ran on GCP VM
- Ran with --mountedDirectory flag
2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

File: tools/integration_tests/local_file/remove_dir_test.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	"path"
+	"testing"
+
+	. "github.com/googlecloudplatform/gcsfuse/tools/integration_tests/local_file/helpers"
+	. "github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/client"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/operations"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
+)
+
+func TestRmDirOfDirectoryContainingGCSAndLocalFiles(t *testing.T) {
```
Comment (vadlakondaswetha) [inline tools/integration_tests/local_file/remove_dir_test.go:28]: remove file will be covered seperately?

File: tools/integration_tests/local_file/helpers/file_operations_helper.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+func ValidateNoFileOrDirError(path string, t *testing.T) {
+	_, err := os.Stat(path)
+	if err == nil || !strings.Contains(err.Error(), "no such file or directory") {
+		t.Fatalf("os.Stat(%s). Expected: %s, Got: %v", path,
+			"no such file or directory", err)
+	}
+}
+
+func VerifyRenameOperationNotSupported(err error, t *testing.T) {
```
Comment (vadlakondaswetha) [inline tools/integration_tests/local_file/helpers/file_operations_helper.go:31]: this is specific to rename. please move this to rename_test.go

File: tools/integration_tests/local_file/helpers/file_operations_helper.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+package helpers
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func ValidateNoFileOrDirError(path string, t *testing.T) {
```
Comment (vadlakondaswetha) [inline tools/integration_tests/local_file/helpers/file_operations_helper.go:23]: Please move this to a more generic helper/util outside local_file tests.

Comment (vadlakondaswetha) [reply tools/integration_tests/local_file/helpers/file_operations_helper.go:23]: Sorry, I missed this during review.

Anything not related to GCS shouldn't be in this file: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/util/client/gcs_helper.go
WritingToLocalFileShouldNotWriteToGCS -> This is specific flow not related to GCS. only operations on GCS has to be in gcs_helper.

You can take it up after all these PRs are merged.

File: tools/integration_tests/local_file/remove_dir_test.go (hunk 1 above)
Comment (ashmeenkaur) [reply tools/integration_tests/local_file/remove_dir_test.go:28]: Yes, I've created a separate PR for it.

File: tools/integration_tests/local_file/helpers/file_operations_helper.go (hunk 3 above)
Comment (ashmeenkaur) [reply tools/integration_tests/local_file/helpers/file_operations_helper.go]: moved NewFileShouldGetSyncedToGCSAtClose and WritingToLocalFileShouldNotWriteToGCS helper methods to local_file tests as they were specific to local file tests. Once all these PRs are merged, I will take another look.

File: tools/integration_tests/local_file/helpers/file_operations_helper.go (hunk 2 above)
Comment (ashmeenkaur) [reply tools/integration_tests/local_file/helpers/file_operations_helper.go]: done

File: tools/integration_tests/util/operations/validate_error.go [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package operations
```
Comment (vadlakondaswetha) [inline tools/integration_tests/util/operations/validate_error.go:1]: rename it to validations_helper.go

**Review by vadlakondaswetha — APPROVED** [review]

**Pull request merged**